packages/mdc-select/component.ts

```diff
@@ -88,20 +84,17 @@ export class MDCSelect extends MDCComponent<MDCSelectFoundation> implements MDCR
       helperTextFactory: MDCSelectHelperTextFactory = (el) => new MDCSelectHelperText(el),
   ) {
     this.isMenuOpen_ = false;
-    this.nativeControl_ = this.root_.querySelector(strings.NATIVE_CONTROL_SELECTOR);
-    this.selectedText_ = this.root_.querySelector(strings.SELECTED_TEXT_SELECTOR);
+    this.selectedText_ = this.root_.querySelector(strings.SELECTED_TEXT_SELECTOR) as HTMLElement;
 
-    const targetElement = this.nativeControl_ || this.selectedText_;
-    if (!targetElement) {
+    if (!this.selectedText_) {
```

Comment:
We usually do not verify for existence of supporting DOM elements. We can remove this check.